include/speed_calc.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SPEED_H
+#define SPEED_H
+
+#include "Arduino.h"
+#include "const.h"
+#include "IOManagement.h"
+#include "STM32TimerInterrupt_Generic.h"
+#include "canPDC.h"
+
+#define INTERVAL_MS 50 // ms
```

Comment:
Let's rename this to be more descriptive, since we have other intervals for other timers. Something like `SPEED_CALC_INTERVAL_MS`. You'll have to change all instances of this.